### PR TITLE
chore(build): Rename leaflet mapping components

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

## Checklist:
- [ ] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [ ] The commit is signed.
- [ ] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

File: packages/marshaller/src/ddl2/model/visualization.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export type VizType = "Table" | "FieldForm" |
 const VizTypeMap: { [key: string]: new (...args: any[]) => {} } = {
     Table, FieldForm,
     Area, Bubble, Bar, Column, Contour, HexBin, Line, Pie, Radar, RadialBar, Scatter, Step, WordCloud,
-    USCountiesChoropleth: ChoroplethCounties, USStatesChoropleth: ChoroplethStates, ClusterPins,
+    USCountiesChoropleth: ChoroplethCounties, USStatesChoropleth: ChoroplethStates, ClusterPins: Leaflet.ClusterPins,
```
Comment (jbrundage) [inline packages/marshaller/src/ddl2/model/visualization.ts]: No support for Leaflet.Circles or paths here?

**Comment by jbrundage** [comment]
That said, the abstraction of Circles etc into projected layers could be added as an afterthought. Otherwise, looks good.

File: packages/marshaller/src/ddl2/model/visualization.ts (hunk 1 above)
Comment (GordonSmith) [reply packages/marshaller/src/ddl2/model/visualization.ts]: Not yet - wanted to just introduce one.  The namespace change shouldn't change anything from the Dashy point of view (apart from this line)

**Comment by GordonSmith** [comment]
FWIW the reason I added the namespace was to not have a Pins + Pins2, but rather Pins + Leaflet.Pins

**Pull request merged**